src/test/kotlin/nl/info/client/brp/util/BRPClientHeadersFactoryTest.kt

```diff
@@ -20,6 +20,7 @@ import java.util.Optional
 class BRPClientHeadersFactoryTest : BehaviorSpec({
     val apiKey = "apiKey"
     val originOin = "originOin"
+    val testValue = "test"
```

Comment:
```suggestion
    val process = "process"
```
Name the value after the parameter, just like the others.
I would even prefix them all with `test`, but it's also fine to stick with the current pattern.